content/en/blog/_posts/2024-06-19-Honor-PV-Reclaim-Policy-Beta.md

```diff
@@ -0,0 +1,193 @@
+---
+layout: blog
+title: 'Kubernetes 1.31: Prevent PersistentVolume leaks when deleting out of order'
+date: 2024-XX-XX
+slug: kubernetes-1-31-prevent-persistentvolume-leaks-when-deleting-out-of-order
+author: >
+  Deepak Kinni (Broadcom)
+---
+
+[PersistentVolume](/docs/concepts/storage/persistent-volumes/) (or PVs for short) are
+associated with [Reclaim Policy](https://kubernetes.io/docs/concepts/storage/persistent-volumes/#reclaim-policy).
+The Reclaim Policy is used to determine the actions that need to be taken by the storage
+backend on deletion of the PV.
```

Comment:
This could be read as the action is taken _when_ a PV is deleted, while it's PVC. I can see you have it correctly below, not sure if it's worth rephrasing here.

Reply:
Updated.